EventHubs/src/EventHubs_Logs/lib/datatransformer.js

```diff
@@ -67,30 +65,56 @@ Transformer.prototype.getProperties = function (resourceGroupName) {
  * @returns {object} of log following GCM's standard format
  */
 Transformer.prototype.generateFormattedLog = function (context, msg) {
-    var properties = this.getProperties(this.getValue(() => msg.properties.resource.resourceGroupName, ''));
+    // Logs from Azure Function follow different format than logs from Logic App
+    if(msg.category == 'FunctionAppLogs') {
+        // There is no resource group field in the message, but does RG name is included with resource id
+        var properties = this.getProperties(this.getValue(() => msg.resourceId.split('/')[4], ''));
```

Comment:
We should add a comment with a sample resource id, so as to exemplify why RG is the 4th segment.  This would help troubleshooting later on.